next-i18next.config.js

```diff
@@ -24,5 +24,7 @@ module.exports = {
     // saveMissing: false,
     // strictMode: true,
     // serializeConfig: false,
-    react: { useSuspense: false }
+    react: { useSuspense: false },
+
+    returnObjects: true
```

Comment:
Curious, what does this do?

Reply:
So it's supposed to be a config option, but I actually I don't think it belongs there, it really is used like this: `t("content.meet.content3", {returnObjects: true})` And basically what it does is if you have nested objects in your translation JSON file, you can have i18n return the object itself rather than just a plain string if returnObjects is set to true. So it wont be needed very often but its handy for nested objects.
I think(?) it can be deleted from this file but I'm not 100% sure.

Reply:
Will need to test this, but I think enabling this will allow us to return objects, say if we had an array or something. Maybe this sets it to true by default? If so, not required here in this file.